app-emulation/spice-protocol/spice_protocol-0.14.4.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="SPICE protocol definitions"
+DESCRIPTION="Headers defining the SPICE-protocol"
+HOMEPAGE="https://www.spice-space.org/"
+COPYRIGHT="2009 Red Hat, Inc"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://www.spice-space.org/download/releases/spice-protocol-$portVersion.tar.xz"
+CHECKSUM_SHA256="04ffba610d9fd441cfc47dfaa135d70096e60b1046d2119d8db2f8ea0d17d912"
+SOURCE_DIR="spice-protocol-$portVersion"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```

Comment:
The recipe doesn't seem to support secondary architectures. Or should it be 'any'?

Reply:
This recipe only includes headers for spice-protocol, it should work for all the platform.

Reply:
Hmm good question, headers on x86 should be in headers/x86 I suppose.
@Begasus what do you think?

Reply:
There are some other packages with headers only files, afaik they don't depend on secondary architecture and should be any, if there is a test case or some other package depending on the headers it could be checked against it (got some time tomorrow (I hope) to look into this)

Reply:
I have fix the spice-protocol's archtectures to **any**